Correct upgrade instructions (#194)
anbox-cloud/discourse-docs: howto/update/upgrade-anbox.md

> - Starting with the 1.14 release, all charms come from [Charmhub](https://charmhub.io) and use the concept of [channels](https://snapcraft.io/docs/channels) to track particular versions. The instructions below address how to upgrade from a 1.13.x release, where charms were still from the old Juju charm store. The `--switch --channel=1.14/stable` arguments instruct Juju to switch to the [Charmhub](https://charmhub.io) version of the charm and track the right channel.

Reviewer: ```suggestion
- Starting with the 1.14 release, all charms come from [Charmhub](https://charmhub.io) and use the concept of [channels](https://snapcraft.io/docs/channels) to track particular versions. The instructions below address how to upgrade from a 1.13.x release, where charms were still from the old Juju charm store. The `--channel=1.16/stable` arguments instruct Juju to switch to the [Charmhub](https://charmhub.io) version of the charm and track the right channel.
```

Reviewer: Is that comment still valid? I see you removed the `--switch`?

Author: Yes, the comment is still valid but the `--switch` is not needed

> - With the 1.14 release the name of the `lxd` charm has changed to `ams-lxd`. If you run a deployment older than 1.14 and want to upgrade you need to upgrade by adding `--switch=ams-lxd` to the upgrade command to make Juju switch to the new charm. The charm itself remains identical with the same functionality and features.

Reviewer: ```suggestion
- With the 1.14 release, the name of the `lxd` charm changed to `ams-lxd`. If you run a deployment older than 1.14 and want to upgrade, add `--switch=ams-lxd` to the upgrade command to make Juju switch to the new charm. The charm itself remains identical with the same functionality and features.
```
